neural_compressor/model/onnx_model.py

```diff
@@ -674,6 +682,34 @@ def find_qkv_in_attention(self, find_all=False):
                     break
         return qkv
 
+    def find_ffn_matmul(self, attention_index, attention_matmul_list, block_len):
+        """Find MatMul in FFN.
+
+        Args:
+            attention_index (list): index of Attention
+            attention_matmul_list (list): list of Attention and MatMul nodes
+            block_len (int): block length
+
+        Returns:
+            list: list of MatMul in FFN
+        """
+        ffn_matmul = []
+        for idx in range(len(attention_index)):
+            if idx != len(attention_index) - 1:
+                index = attention_index[idx + 1]
+                if index - 2 >= 0 and index - 1 >= 0:
+                    ffn_matmul.append([attention_matmul_list[index - 2], attention_matmul_list[index - 1]])
+            else:
+                index = attention_index[idx]
+
+                if index + block_len - 2 < len(attention_matmul_list) and index + block_len - 1 < len(
```

Comment:
` index + block_len - 2 < index + block_len - 1`

Reply:
Thanks. Fix in [25b48b3](https://github.com/intel/neural-compressor/pull/1197/commits/25b48b364de2bd09d632ab16477eff3a006ef936)